app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+from __future__ import annotations
```

Comment:
The import statement `from __future__ import annotations` should be placed at the top of the file, without any leading spaces. Ensure there are no spaces before this line to avoid syntax errors.